streets_reader/src/split_ways.rs

```diff
@@ -118,13 +117,6 @@ pub fn split_up_roads(
                 continue;
             }
             if let Some(i2) = pt_to_intersection.get(&pt.to_hashable()) {
-                if i1 == endpt1 {
```

Comment:
This was a hack to remember if a segment is at the beginning or end of an OSM way, so that turn lane restrictions can be interpreted sanely